src/main/java/bio/terra/cloudfunctions/common/CloudEventsHarness.java

```diff
@@ -0,0 +1,102 @@
+package bio.terra.cloudfunctions.common;
```

Comment:
I don't think we need a superclass for the cloud function implementations just yet.

Reply:
Let's revisit the specification of the harness as follows:

- Framework-specific logic to receive and parse messages
- Currently handles Cloud Function events (in the future the harness could be replaced by a Spring Integration-specific class and run inside k8s)
- Contains no business logic

The `CloudEventsHarness` met the above specification, any specific variant of a harness is going to satisfy the above specification so you would need the same basic logic as in `CloudEventsHarness` one way or another.